src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/EventValidationService.java

```diff
@@ -191,6 +198,41 @@ public List<String> validateHearingNumber(CaseData caseData, CorrespondenceType
     public void validateJurisdictionCodes(CaseData caseData, List<String> errors) {
```

Comment:
Maybe it is worth renaming this to validateJurisdiction as it now also encompasses validating the date